elements/pf-text-input/pf-text-input.ts

```diff
@@ -213,48 +228,78 @@ export class PfTextInput extends LitElement {
   }
 
   override render() {
+    const { valid } = this.#internals.validity;
     return html`
       <input id="input"
+             .placeholder="${this.placeholder ?? ''}"
+             .value="${this.value}"
+             .pattern="${this.pattern as string}"
              @input="${this.#onInput}"
+             @blur="${this.#onBlur}"
              ?disabled="${this.matches(':disabled') || this.disabled}"
              ?readonly="${this.readonly}"
              ?required="${this.required}"
              aria-label="${this.#derivedLabel}"
-             placeholder="${ifDefined(this.placeholder)}"
              type="${ifDefined(this.type)}"
-             .value="${this.value}"
              style="${ifDefined(this.customIconUrl && styleMap({
                backgroundImage: `url('${this.customIconUrl}')`,
                backgroundSize: this.customIconDimensions,
              }))}">
+        <span id="helper-text" ?hidden="${!this.helperText ?? valid}">${this.helperText}</span>
```

Comment:
If you specify helper text and error text, should only the error text show if the field is invalid? They both display at the moment. I couldn't find an example that had both in the PF4 docs either.

Reply:
Maybe they should both show? Sure the helper applies in all cases. 
Let's merge and fix before release it it's a problem